### PR TITLE
Add requester permissions

### PR DESCRIPTION
**Pull request by a-belhadj**

### CODE REVIEW

File: tests/test_profiles/test_model/test_has_perm_for_owner.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    def _assert_user_can(self, action, user, obj):
+        print(f"testing that {user} can {action} on {obj}")
+        self.assertTrue(user.has_perm(self._get_action_perm(obj, action), obj))
+
+    def _assert_user_cant(self, action, user, obj):
+        print(f"testing that {user} cannot {action} on {obj}")
+        self.assertFalse(user.has_perm(self._get_action_perm(obj, action), obj))
+
+    def __test_has_perm_generic_with_requester_perm_user_permission(self, obj, action):
```
Comment (EliasBoulharts) [inline tests/test_profiles/test_model/test_has_perm_for_owner.py:55]: 2x _

**Review by EliasBoulharts — CHANGES_REQUESTED** [review]
change requesterpermission to ownerpermission and rename user field in request to requester + generate a new migration

File: tests/test_profiles/test_model/test_get_queryset_instance.py [hunk 2]
```diff
@@ -37,9 +37,34 @@ def _assert_can_see_nothing(self, user):
...
+        self._assert_can_see_nothing(self.superuser)
+        self._assert_can_see_nothing(self.user1)
+
+        # create a new instance
+        instance1 = Instance.objects.create(name="Instance #1", quota_scope=self.default_quota_scope,
+                                            requester=self.user1)
+        self.assertTrue(instance1 in Instance.objects.all())
+        self.assertEqual(Instance.objects.count(), 1)
+
+        # add global perm
```
Comment (Sispheor) [inline tests/test_profiles/test_model/test_get_queryset_instance.py:51]: owner perm

File: tests/test_profiles/test_model/test_get_queryset_request.py [hunk 3]
```diff
@@ -52,9 +52,60 @@ def _assert_can_see_nothing(self, user):
...
+        self._assert_can_see_nothing(self.user1)
+
+        # create a new request
+        instance1 = Instance.objects.create(name="Instance #1", quota_scope=self.default_quota_scope,
+                                            requester=self.user1)
+        request1 = Request.objects.create(instance=instance1, operation=self.operation)
+        self.assertTrue(request1 in Request.objects.all())
+        self.assertEqual(Request.objects.count(), 1)
+
+        # add global perm
```
Comment (Sispheor) [inline tests/test_profiles/test_model/test_get_queryset_request.py:67]: owner

File: tests/test_profiles/test_model/test_get_queryset_support.py [hunk 4]
```diff
@@ -37,9 +37,68 @@ def _assert_can_see_nothing(self, user):
...
+        self._assert_can_see_nothing(self.user1)
+
+        # create a new request
+        instance1 = Instance.objects.create(name="Instance #1", quota_scope=self.default_quota_scope,
+                                            requester=self.user1)
+        request1 = Support.objects.create(instance=instance1)
+        self.assertTrue(request1 in Support.objects.all())
+        self.assertEqual(Support.objects.count(), 1)
+
+        # add global perm
```
Comment (Sispheor) [inline tests/test_profiles/test_model/test_get_queryset_support.py:56]: owner

**Review by EliasBoulharts — APPROVED** [review]

**Review by Sispheor — APPROVED** [review]

**Pull request merged**